mobile/library/cc/stream.h

```diff
@@ -21,8 +21,10 @@ class Stream {
    *
    * @param headers the headers to send.
    * @param end_stream indicates whether to close the stream locally after sending this frame.
+   * @param idempotent indicates that the request is idempotent. When idempotent is set to true
+   *                   Envoy Mobile may perform retry on failures. By default, it is set to false.
```

Comment:
can we qualify
HTTP/3 post-handshake failures
?
We already retry on failures automatically for safe requests for example

Reply:
Done.